lib/fizzy/instantiate.cpp

```diff
@@ -470,18 +471,23 @@ std::optional<FuncIdx> find_exported_function(const Module& module, std::string_
     return find_export(module, ExternalKind::Function, name);
 }
 
+ExecutionResult ExecuteFunction::operator()(Instance& instance, const Value* args, int depth)
```

Comment:
Nit: this feels like a weird place for this. Aren't also these functions within an anonymous namesapce?

Reply:
This is `fizzy` namespace, unnamed ends at line 303.

It can't be in header, because it uses `execute`, that's why it's in this file.

Reply:
I mean just put it up on the top of the file or so.

Reply:
Sure, the main point is not to have it intermingled into helpers.